runtime/src/tests.rs

```diff
@@ -560,43 +560,36 @@ mod pallets_interact {
 mod payout {
     use pallet_staking::EraPayout;
 
-    use crate::{Balance, Runtime, CENTS};
+    use crate::{Balance, Runtime};
 
     use super::new_test_ext;
 
-    /// Test that validators receive a cumulative reward that mimics the current emission of
-    /// $ZEN in the PoW Horizen blockchain for miners, which is a coinbase of 6.25 Zen for
-    /// each block every 2.5 minutes.
+    #[ignore]
```

Comment:
Maybe it's better to remove the `ignore`.

Reply:
Done thanks